bin/local-env/start.sh

```diff
@@ -23,9 +23,6 @@ cd "$(dirname "$0")/../.."
 # done on every time `npm run test-e2e` is run.
 . "$(dirname "$0")/install-wordpress.sh"
 
-# Install specific version of Gutenberg plugin for e2e tests.
-download_gutenberg "14.7.3" "tests/e2e/plugins/gutenberg.14.7.3.zip"
```

Comment:
Just in case we ever need it, should we leave the `download_gutenberg` util function in place?

Reply:
Same for `uploads.ini`

https://github.com/ampproject/amp-wp/blob/96cdfb90e4124ccfc5f99c46d006a8a564479b4f/bin/local-env/docker-compose.yml#L13

Reply:
We could, but probably better to remove it. It can be resurrected via version control if needed.